cockatrice/src/dlg_forgotpassword.cpp

```diff
@@ -0,0 +1,67 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpassword.h"
+
+DlgForgotPassword::DlgForgotPassword(QWidget *parent)
+    : QDialog(parent)
+{
```

Comment:
Im on @Daenyth side on this one (which generally doesn't happen).  Get the majority to agree with you and we can change it.  Im against the idea for now.  Im more in favor of making sure the user types the info in forcing a self aware verification type process.  But thats just my $0.02.